mapie/classification.py

```diff
@@ -292,9 +303,18 @@ def _check_cv(
         """
```

Comment:
Please update the whole docstring to the new code

Reply:
See above.

Reply:
Docstring updated